sdk/packages/analytics/src/e2e.test.ts

```diff
@@ -339,7 +337,7 @@ describe('Analytics client tests', () => {
     expect(response.status).toBe(403)
   })
 
-  describe('Test events', () => {
+  describe.concurrent('Test events', () => {
```

Comment:
do we want this?

Reply:
It would make it closer to the real use case, speeds up tests as well